doc/FAQ.md

```diff
@@ -74,7 +74,9 @@ Other nodes might not be able to connect to you, but your node can still connect
 But Core Lightning also has an integrated IPv4/6 address discovery mechanism.
 If your node detects an new public address, it will update its announcement.
 For this to work binhind a NAT router you need to forward the default TCP port 9735 to your node.
-IP discovery is only active if no other addresses are announced.
+
+Note: Per default IP discovery is only active if no other addresses are announced.
+This can also be explicitly activated using the `--ip-discovery=true` switch.
```

Comment:
What would IP discovery be used if we don't announce addresses? I assume it is still displayed in `getinfo` for example.

Reply:
If ip discovery is active (implicitly or explicitly) it announces discovered addresses and shows them via `getinfo`.

What I tried to explain is that IP discovery is almost always deactivated, unless your node wasn't even able to discover a local IPv6 or TOR address or you told him by disabling auto announcement, i.e. via `--bind=""`.

Maybe I can re-write these lines